Updates for OCP 3.6 install/upgrade/downgrade

adellape opened the pull request:
Preview (internal):

http://file.rdu.redhat.com/~adellape/080317/ocp36_installupgrade/install_config/install/planning.html

adellape commented:
Still need to update `manual_upgrades`.

adellape commented:
![](https://github.trello.services/images/mini-trello-icon.png) [General updates for OCP 3.6 install & upgrade](https://trello.com/c/yUTXiAvc/609-general-updates-for-ocp-36-install-upgrade)

jiajliu commented:
@adellape 
We need not install/update/exclude/unexclude both atomic-openshift-excluder and atomic-openshift-docker-excluder manually during install/upgrade openshift from in 3.6, because all things have been done automatically by openshift-ansible. This function has been supported from 3.5 and kept in 3.6, and backported into 3.3&3.4 now.

adellape commented:
> We need not install/update/exclude/unexclude both atomic-openshift-excluder and atomic-openshift-docker-excluder manually during install/upgrade openshift from in 3.6, because all things have been done automatically by openshift-ansible. This function has been supported from 3.5 and kept in 3.6, and backported into 3.3&3.4 now.

@jiajliu Thanks, I've removed mention of the excluder packages from the Host Preparation, Advanced Installation, and Automated Upgrades topics; those changes can be seen separately in commit https://github.com/openshift/openshift-docs/pull/4934/commits/73c88a61971d9d717a27c9bfef2e88a962a18aed. Note that I have not removed them from the manual upgrade steps since they would not be able to rely on the playbook to handle the excluding. Also I added the fuller version number to `yum install docker-1.12.6` in the Host Preparation topic's "Installing Docker" section, since they wouldn't have the excluder in place yet (pre-installation) to lock them to the correct version.

jiajliu commented:
@adellape 
A little issue about excluder in quick install section that no need run "atomic-openshift-excluder exclude" after installation finished. Other updates LGTM.

adellape commented:
> A little issue about excluder in quick install section that no need run "atomic-openshift-excluder exclude" after installation finished. Other updates LGTM.

@jiajliu Thank you for review. I can't find a reference to the excluder in the Quick Installation topic; are you seeing it in the source or the preview build?

ahardin-rh commented:
LGTM! ⭐️

bmcelvee commented:
LGTM too! 🐱

jiajliu commented:
@adellape 
I'm not quite sure in which link i was seeing it, anyway I found no excluder in latest Quick Installation topic from following doc. So anything looks good now. http://file.rdu.redhat.com/~adellape/080317/ocp36_installupgrade/install_config/install/quick_install.html.
